src/libstd/os/raw/char.md

```diff
@@ -0,0 +1,11 @@
+Equivalent to C's `char` type.
+
+[C's `char` type] is completely unlike [Rust's `char` type]; while Rust's type represents a unicode scalar value, C's `char` type is just an ordinary integer. In practice, this type will always be either [`i8`] or [`u8`], but you're technically not supposed to rely on this behaviour, as the standard only defines a char as being at least eight bits long.
+
+C chars are most commonly used to make C strings. Unlike Rust, where the length of a string is included alongside the string, C strings mark the end of a string with a zero. See [`CStr`] for more information.
```

Comment:
>C strings mark the end of a string with a zero

This is slightly misleading as someone might think it means `0` the character, not the literal integer value 0.

Reply:
Would clarifying "the character '\0' " be sufficient?